frontend/src/app/oe/components/strike-summary/strike-summary.component.ts

```diff
@@ -74,6 +74,10 @@ export class StrikeSummaryComponent implements OnInit, OnDestroy {
     return this.chainworkDiff / BigInt(this.timeDiff);
   }
 
+  get strikeDetailLink(): string {
+    return this.relativeUrlPipe.transform(`/hashstrikes/strike_detail/${this.fromBlock.height}/${this.toBlock.height}/${this.toBlock.height}/${this.fromBlock.mediantime}/${this.toBlock.mediantime}`);
```

Comment:
`${this.toBlock.height}/${this.toBlock.height}` two times block height?

Reply:
Yes, those are intended to satisfy the structural needs of the strike detail component